src/test/kotlin/io/github/gradlenexus/publishplugin/TaskOrchestrationTest.kt

```diff
@@ -135,6 +191,34 @@ class TaskOrchestrationTest {
             .contains("otherNexus")
     }
 
+    @Test
+    internal fun `description of simplified close task contains names of all defined Nexus instances`() {
+        initSingleProjectWithDefaultConfiguration()
+        project.extensions.configure<NexusPublishExtension> {
+            repositories.create("otherNexus")
+        }
+
+        val closeTask = getJustOneTaskByNameOrFail(NexusPublishPlugin.SIMPLIFIED_CLOSE_TASK_NAME)
+
+        assertThat(closeTask.description)
+            .contains("sonatype")
+            .contains("otherNexus")
+    }
+
+    @Test
+    internal fun `description of simplified release task contains names of all defined Nexus instances`() {
+        initSingleProjectWithDefaultConfiguration()
+        project.extensions.configure<NexusPublishExtension> {
+            repositories.create("otherNexus")
+        }
+
+        val releaseTask = getJustOneTaskByNameOrFail(NexusPublishPlugin.SIMPLIFIED_RELEASE_TASK_NAME)
+
+        assertThat(releaseTask.description)
+            .contains("sonatype")
+            .contains("otherNexus")
+    }
```

Comment:
The same here.

Reply:
In that case with more parameter, `@CsvSource` might be more useful, especially the new `textBlock` parameter which tries to mimic a table from Spock:
https://junit.org/junit5/docs/current/api/org.junit.jupiter.params/org/junit/jupiter/params/provider/CsvSource.html#textBlock()

Reply:
Sorry, seen too late, I now used a `@MethodSource` but I can change it if you want.
Anyway, I'm now reminded of one point why I prefer Spock. :-D